README.md

```diff
@@ -69,10 +69,10 @@ This fork is intended as temporary measure and will possibly stop being maintain
 ### Install from source
 
  - clone this repo
- - install py3 and pip if needed (following instruction are for apt-based distributives)
- - `sudo apt-get update`
- - `sudo apt-get install python3-pip`
- - `sudo python3 -m pip install -r requirements.txt`
+ - install python3 and pip if needed (following instruction are for apt-based distributives)
+   - `sudo apt-get update`
+   - `sudo apt-get install python3-pip`
```

Comment:
Note: you can also use just `apt` instead of `apt-get`.

Reply:
done , thx!